.env

```
@@ -0,0 +1,4 @@
+TMG_HOST = 'localhost'
+TMG_USER = 'root'
+TMG_DB = 'test-schema'
+TMG_CHARSET = 'utf8mb4'
```

```suggestion
TMG_HOST=
TMG_USER=
TMG_DB=
TMG_CHARSET=
```
ここは各自で設定するとことだからこれでええぞ

---

.gitignoreに.env入れてもいいかもな